src/e2e/iOS/__tests__/caret.ts

```diff
@@ -104,3 +105,30 @@ it('Tap hidden root thought', async () => {
   const editingText = await getEditingText(mobileBrowser)
   expect(editingText).toBe('b')
 })
+
+it('Tap hidden uncle', async () => {
+  const importText = `
+    - a
+      - b
+        - c
+      - d`
+  await gesture(mobileBrowser, gestures.newThought)
+  await paste(mobileBrowser, [''], importText)
+  await clickThought(mobileBrowser, 'a')
+  await clickThought(mobileBrowser, 'b')
+  await clickThought(mobileBrowser, 'c')
+
+  const editableNodeHandle = await waitForEditable(mobileBrowser, 'c')
+  const elementRect = await getElementRectByScreen(mobileBrowser, editableNodeHandle)
+
+  // click near uncle. see: https://github.com/cybersemics/em/pull/1128#issuecomment-837427219
+  await mobileBrowser.touchAction({
+    action: 'tap',
+    x: elementRect.x + (elementRect.width / 2),
+    y: elementRect.y + elementRect.height + 10,
+  })
```

Comment:
Could you do this with `tapWithOffset`? `getElementRectByScreen` would be handy in other situations, but maybe here we can use what we already have. It would be nice to abstract away the pixel calculations anyway.